scripts/globals/abilities/quickstep.lua

```diff
@@ -174,7 +174,12 @@ ability_object.onUseAbility = function(player, target, ability, action)
         ability:setMsg(xi.msg.basic.JA_MISS)
     end
 
-    action:setAnimation(target:getID(), getStepAnimation(player:getWeaponSkillType(xi.slot.MAIN)))
+    if player:getName() == "uka_totlihn" then
```

Comment:
can we also add an entity type check to ensure the user is a trust for this conditional? I know normally you can't input that as a name but there are ways to pull it off. and hopefully in the future non player used abilities have their own place or the situations requiring conditionals cease to exist.